bin/reth/src/args/network_args.rs

```diff
@@ -88,12 +88,8 @@ impl NetworkArgs {
 
         // Configure peer connections
         let mut peer_config = config.peers.clone();
-        if let Some(max_inbound_peers) = self.max_inbound_peers {
-            peer_config = peer_config.with_max_inbound(max_inbound_peers);
-        }
-        if let Some(max_inbound_peers) = self.max_outbound_peers {
-            peer_config = peer_config.with_max_outbound(max_inbound_peers);
-        }
+        peer_config = peer_config.with_max_inbound_opt(self.max_inbound_peers);
+        peer_config = peer_config.with_max_outbound_opt(self.max_outbound_peers);
```

Comment:
nit: these can be chained:

`let peer_config = config.peers.clone().with_().with()`

Reply:
Thanks for the review, updated!